### PR TITLE
[bitnami/harbor] Remove references to harbor notary server and harbor notary signer

### LINKED ISSUE #19248: [bitnami/harbor] Upgrade Harbor to version 2.9
### Name and Version

bitnami/harbor

### What is the problem this feature will solve?

Currently, the latest release of Harbor isn't supported.  Harbor 2.9 introduces some breaking changes by removing Notary.

### What is the feature you are proposing to solve the problem?

Update Harbor and remove Notary from its components.

### What alternatives have you considered?

_No response_

### PR DESCRIPTION
**Pull request by Mauraza**
### Description of the change

The new version of harbor was deprecated harbor notary

### Applicable issues

- fixes #19248

### Checklist

- [X] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/). This is *not necessary* when the changes only affect README.md files.
- [X] Variables are documented in the values.yaml and added to the `README.md` using [readme-generator-for-helm](https://github.com/bitnami-labs/readme-generator-for-helm)
- [X] Title of the pull request follows this pattern [bitnami/<name_of_the_chart>] Descriptive title
- [X] All commits signed off and in agreement of [Developer Certificate of Origin (DCO)](https://github.com/bitnami/charts/blob/main/CONTRIBUTING.md#sign-your-work)

### CODE REVIEW

File: bitnami/harbor/README.md [hunk 1]
```diff
@@ -1201,6 +1048,11 @@ Find more information about how to deal with common errors related to Bitnami's
 
 ## Upgrading
 
+### To 18.0.0
+
+This major update the harbor notary server and harbor notary signer are deprecated. This was depreacates in the version 2.9.0 of harbor
+[Here](https://github.com/goharbor/harbor/wiki/Harbor-Deprecates-Notary-v1-Support-in-v2.9.0)
```
Comment (juan131) [inline bitnami/harbor/README.md:1054]: There are some typos in this sentence:
```suggestion
This major deprecates harbor notary server and harbor notary signer components. These components were deprecated in Harbor 2.9.0, find more information in [Harbor wiki](https://github.com/goharbor/harbor/wiki/Harbor-Deprecates-Notary-v1-Support-in-v2.9.0).

**Review by juan131 — CHANGES_REQUESTED** [review]
I saw you remove some pieces on `conf/notary-server.json` & `conf/notary-signer.json`. Do we still need these files at all?

**Comment by Mauraza** [comment]
> I saw you remove some pieces on `conf/notary-server.json` & `conf/notary-signer.json`. Do we still need these files at all?

I don't think so because they are related to harbor-notary, which is deprecated in the harbor.

**Comment by juan131** [comment]
> I don't think so because they are related to harbor-notary, which is deprecated in the [harbor](https://github.com/goharbor/harbor/wiki/Harbor-Deprecates-Notary-v1-Support-in-v2.9.0).

Then, we should completely remove them, right?

**Review by juan131 — APPROVED** [review]
LGTM

**Pull request merged**